projects/core/src/lib/image-resizer.ts

```diff
@@ -82,7 +82,7 @@ function resize(image: HTMLImageElement, method: ResizeMethod, maxSize: ImageSiz
  * Convert image to canvas
  */
 function imageToCanvas(image: HTMLImageElement, width: number, height: number): HTMLCanvasElement {
-  const canvas = document.createElement('canvas');
+  const canvas = <HTMLCanvasElement> document.createElement('canvas');
```

Comment:
This caused the build to fail in Angular 7, while it seems to work fine with Angular 6.